content/_releases/22/22.md

```diff
@@ -0,0 +1,235 @@
+
+### Email domain blacklist
+
+If your tenant has an an open registration page then you can block registration
+from any domain by adding it into the domain blacklist. For this to work in your
+tenant, make an [API call]({{site.data.tenant.apiDocsUri}}/v2/#update-a-tenant)
+to update your tenant records using `tenantAdmin` privileges:
+
+*   Enable the registration page - set the `attributes.hide_register` parameter to `false`.
+*   Add records to the blacklist - create the `attributes.email_domains_blacklist` record to contain comma separated domain names in an array object like `["example.com", "foo.bar"]`.
+
+### Email verification with regular signup page
+
+During the 20.20 platform release we introduced [email verification process](/releases/22/20#email-verification-during-registration) which worked only when the [`signin_v_2` were used](/releases/22/20#setup-email-verification).
+
+Now you can use the email verification process even if you are using a regular
+signup page in your tenant. To enable this feature in your tenant the following
+steps must be done using an [API call]({{site.data.tenant.apiDocsUri}}/v2/#update-a-tenant)
+to update your tenant records using `tenantAdmin` privileges:
+
+*   Enable the registration page - set the `attributes.hide_register` parameter to `false`.
+*   Enable `email_verification` feature flag - set `attributes.feature_flags.email_verification` to `true`.
+
+{% endif %}
+
+## Fixed bugs
+
+*   Fixed the user interface bug when the Execution page would not load for some time when the workspace contained execution of a flow who's author deleted his/her account permanently. Now you will see `[Deleted User]` instead.
+*   Addressed the bug when `"Cannot perform 'get' on a proxy that has been revoked"` error was shown after evaluation of the JSONata mapping expression.
+*   Fixed the user interface bug connected with the Dashboard graph when the horizontal line and values would sometimes overlap.
+*   Addressed the problem when the registration page would sometimes create an empty contract even if the user would already exist in the Database.
+*   Fixed the user interface bug connected with the loading a flow versions page when the aggregated document size would exceed 16MB limit of MongoDB.
+
+## Components
+
+### [REST API V2 component](/components/rest-api/)
+
+*   **IMPROVED** the use of platform secrets
+
+### [HJson component](/components/hjson/) `1.0.2`
+
+*   **FIXED** the attachments URL
+
+### [Delta-detection Component](components/delta-detection/) `2.1.0`
+
+*   **ADDED** `Write Entire Bucket Contents` action
+*   **ADDED** `Read Entire Bucket Contents` action
+
+### [utility component](/components/utility/) `1.4.2`
+
+*   **ADDED** `Network Diagnostics` action
+
+### [Ipaas component](/components/ipaas/) `1.4.0`
+
+*   **FIXED** issue - hydrate flow with `pubSubTopicName` at actions `Lookup Object By Unique Criteria` and `Lookup Objects (plural)`
```

Comment:
hydrate? Was the flow drinking too much?

I suggest using another word or adding an explanation in parentheses at first use